run-tests-with-build-drools-jitting-parent/drools-jitting-project-sources-test/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>run-tests-with-build-drools-jitting-parent</artifactId>
+    <groupId>org.kie</groupId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>pom</packaging>
+
+  <artifactId>drools-jitting-project-sources-test</artifactId>
+
+  <properties>
+    <project1>!drools-examples</project1>
+    <project2>!kie-pmml</project2>
+    <maven.additional.options>-Ddrools.jittingThreshold=0 -pl ${project1},${project2}</maven.additional.options>
```

Comment:
Additional options necessary for Drools Jitting tests. It was also necessary to remove modules from the build (-pl option) due to a limitation found on Exec Maven Plugin which I was unable to solve. Two of the Drools modules are already using the plugin and don't recognize the parameters from run-tests-with-build project.